chapters/Index.md

```diff
@@ -35,24 +35,2394 @@ Jintek           <info@jintek.consulting>
+
+#### Consensus Splits in Implementation can Destroy Non-Repudiation
+
+Because ::Non-Repudiation:: requires following a complete stream of updates to a DID, any consensus forks in which DID updates to apply can be used to permanently diverge the history for a DID, and thus the key material, creating alternate attestation histories.  As a concrete example, a seemingly small difference between two clients in interpretation of the specification could be used fraudulently to sign a loan in one history and claim that it was never signed in another history.
+
+In order to prevent consensus splits, **did:btcr2** needs a particularly good test suite. It MUST be designed to challenge all the foreseeable edge cases as well as maintained to find new ones.
+
+### Deployment Considerations
+
+#### Update Payloads Stored in Content Addressable Storage (CAS) Systems are Publicly Accessible
+
+Update payloads stored in ::Content Addressable Storage:: (CAS) such as IPFS SHOULD be considered public. Anyone MAY retrieve this information (assuming they have the ::CID::) and use it to track the DID over time. IPFS node operators would have access to all content stored on IPFS and could choose to index certain data like ::BTCR2 Updates::, including the updates posted by that DID's ::BTCR2 Beacon::. This MAY advertise information about the DID that the controller wishes to remain private.
+
+Those parties most concerned about privacy SHOULD maintain their ::DID Update Payloads:: in a ::Sidecar:: manner and provide them to necessary parties during resolution. It is RECOMMENDED not to include any sensitive data other than the necessary DID update data.
+
+#### Beacon Coordinators Know the DIDs Being Aggregated by a Cohort
+
+Within ::Sparse Merkle Tree:: (SMT) ::BTCR2 Beacons::, the DID is used as a path to a leaf node in the ::SMT::. The coordinator MUST know these paths for them to be able to construct the tree and generate the correct proof paths. Within ::Content Identifier:: (CID)-based ::BTCR2 Beacons::, the coordinator MUST construct an aggregated bundle that includes all DIDs aggregated as a key to the ::CID:: for that ::DID's Update Payload::. This means that for both types of ::Aggregate Beacons::, the coordinator necessarily MUST know all DIDs being aggregated by a cohort.
+
+#### CIDAggregate Cohort Members Know All DIDs that are Updated
+
+Cohort members participating in a CIDAggregate ::BTCR2 Beacon:: learn all DIDs that are updated in each ::Beacon Signal::. This is because they SHOULD verify the contents of the ::Beacon Signal:: before authorizing it and a CIDAggregate ::Beacon Signal:: contains a ::CID:: to an update bundle. An update bundle is a JSON object mapping **did:btcr2** identifiers to ::CID:: values for individual ::BTCR2 Updates::. Each DID controller SHOULD independently retrieve and verify the contents of the update bundle to ensure it contains the expected update for their DID.
```

Comment:
I think all of these considerations can be removed with the addition of the hash of the DID as the key into both the Map and SMT beacons.

We may want to replace this with another consideration for Map Beacons. 

This would be that, all participants on the Cohort see the Beacon Announcement Map containing all the keys for DIDs being updated. Even though these keys are obscured by a hash, if a participant sees a DID in the wild they would be able to tell if it is in the cohort or not.

Reply:
Additionally, if the updates are stored on CAS. Then it is trivial for any participant to find our the DID being updated by fetching the update

Reply:
It won't let me make the suggestion here, but if you do, I'll accept.